cli/main.c

```diff
@@ -157,33 +144,65 @@ int main(int argc, char** argv) {
   // FIXME: this is temp till arg parse implemented.
   PkResult result;
 
-  if (argc == 1) {
+  static const char* const usage[] = {
+    "pocket [option] ... [-c cmd | file | -] [arg] ...",
+    NULL,
+  };
+
+  int debug = 0;
+  int quiet = 0;
+  int version = 0;
+  const char* cmd = NULL;
+
+  struct argparse_option flags[] = {
+    OPT_HELP(),
```

Comment:
This will print and exit the program by the `argparse` library. We need to print it by ourselves. Remove this line and get the help option with `OPT_BOOLEAN('h', "help", ...` and add

```c
if (help) {
  argparse_usage(&argparse);
  return 0;
}
```

Also, the name `flags` isn't correct as it's not an array of flags, it's an array of options. change this to something like `cli_opts`